src/packages/client/src/runtime/utils/printStack.ts

```diff
@@ -51,15 +51,17 @@ export const printStack = ({
     const stack = stackTraceParser.parse(callsite)
     // TODO: more resilient logic to check that it's not relative to cwd
     const trace = stack.find((t, i) => {
-      if (i < 2) {
+      if (i < 3) {
```

Comment:
Looks like a magic number to me 😄

Reply:
It's about how deep the stack can be cut away - we have one more function call in our stack, so we can ignore that level. I'll add a comment.

Reply:
Thanks!